cloudcix_primitives/utils.py

```diff
@@ -75,41 +82,46 @@ def load_pod_config(config_file=None, prefix=4000) -> Dict[str, Any]:
 
     config_data = {
       'raw': None,
-      'processed': None
+      'processed': {}
     }
 
+    config = None
+
     # Load config from config_file
     try:
         with Path(config_file).open('r') as file:
-            config['file'] = json.load(file)
+            config = json.load(file)
     except OSError as e:
-            return False, config_data, messages[prefix + 11] + e.__str__()
+            return False, config_data, ("%d: " % (prefix+11)) + messages[11] + e.__str__()
```

Comment:
```suggestion
            return False, config_data, f'{prefix+11}: {messages[11]} {e.__str__()}'
```

f' strings are way simpler and clean to use.